Adyen/Model/CashAppPayConfiguration.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Configuration for Cash App Pay Component.
+public struct CashAppPayConfiguration {
+
+    /// The URL for Cash App to call in order to redirect back to your application.
+    public let redirectURL: URL
```

Comment:
How is it working with session's returnUrl ?

Reply:
Hmm, curious. Let's discuss this in person